specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/datafactory.json

```diff
@@ -2448,1454 +2637,1475 @@
       }
     }
   },
-    "definitions": {
-      "Resource": {
-        "x-ms-azure-resource": true,
-        "description": "Azure Data Factory top-level resource.",
-        "properties": {
-          "id": {
-            "type": "string",
-            "description": "The resource identifier.",
-            "readOnly": true
-          },
-          "name": {
-            "type": "string",
-            "description": "The resource name.",
-            "readOnly": true
-          },
-          "type": {
-            "type": "string",
-            "description": "The resource type.",
-            "readOnly": true
+  "definitions": {

```

Comment:
Most of the differences in "definitions" is format changes.

Reply:
Can you guys eliminate these format changes to make sure important changes are not missed?